app/models/concerns/filter/filter_scopes.rb

```diff
@@ -158,6 +159,83 @@ module Filter::FilterScopes
       report_scope_source.joins(:client).where(c_t[key].eq(1))
     end
 
+    private def filter_for_race_ethnicity_combinations(scope)
+      return scope unless @filter.race_ethnicity_combinations.present?
+
+      race_ethnicity_scope = nil
+      @filter.race_ethnicity_combinations.each do |combination|
+        alternative = case combination.to_sym
+        when :am_ind_ak_native
+          race_ethnicity_alternative(scope, :AmIndAKNative)
+        when :am_ind_ak_native_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :AmIndAKNative, true)
+        when :asian
+          race_ethnicity_alternative(scope, :Asian)
+        when :asian_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :Asian, true)
+        when :black_af_american
+          race_ethnicity_alternative(scope, :BlackAfAmerican)
+        when :black_af_american_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :BlackAfAmerican, true)
+        when :hispanic_latinaeo
+          race_ethnicity_alternative(scope, :HispanicLatinaeo)
+        when :mid_east_n_african
+          race_ethnicity_alternative(scope, :MidEastNAfrican)
+        when :mid_east_n_african_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :MidEastNAfrican, true)
+        when :native_hi_pacific
+          race_ethnicity_alternative(scope, :NativeHIPacific)
+        when :native_hi_pacific_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :NativeHIPacific, true)
+        when :white
+          race_ethnicity_alternative(scope, :White)
+        when :white_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :White, true)
+        when :multi_racial
+          race_ethnicity_alternative(scope, :MultiRacial)
+        when :multi_racial_hispanic_latinaeo
+          race_ethnicity_alternative(scope, :MultiRacial, true)
+        when :race_none
+          race_ethnicity_alternative(scope, :RaceNone)
+        end
+        race_ethnicity_scope = add_alternative(race_ethnicity_scope, alternative)
+      end
+
+      scope.joins(:client).merge(race_ethnicity_scope)
+    end
+
+    private def race_ethnicity_alternative(scope, key, hispanic_latinaeo = false)
+      columns = {
+        AmIndAKNative: 0,
+        Asian: 0,
+        BlackAfAmerican: 0,
+        NativeHIPacific: 0,
+        White: 0,
+        HispanicLatinaeo: 0,
+        MidEastNAfrican: 0,
+      }
```

Comment:
I'm hoping we can decrease the proliferation of race (and gender) column names.  Maybe?
```suggestion
      columns = (HudUtility2024.race_fields - [:RaceNone]).map { |k| [k, 0] }.to_h
```